dask_planner/src/sql/table.rs

```diff
@@ -53,26 +53,21 @@ impl TableSource for DaskTableSource {
     ) -> datafusion_common::Result<TableProviderFilterPushDown> {
         let filters = split_conjunction(filter);
         if filters.iter().all(|f| is_supported_push_down_expr(f)) {
-            // TODO this should return Exact but we cannot make that change until we
-            // are actually pushing the TableScan filters down to the reader because
-            // returning Exact here would remove the Filter from the plan
-            Ok(TableProviderFilterPushDown::Inexact)
+            // Push down filters to the tablescan operation if all are supported
+            Ok(TableProviderFilterPushDown::Exact)
```

Comment:
With this change should we be able to remove the `clippy` override for this function?